EZSwiftExtensionsTests/ArrayTests.swift

```diff
@@ -247,4 +247,14 @@ class ArrayTests: XCTestCase {
         XCTAssertEqual(duplicatedElementsArrayOrderMaintained.unique(), [2, 1, 3])
         
     }
+    
+    func testRemoveEqualItems() {
+        var list = ["one", "two", "two", "four"]
+        list.removeEqualItems(item: "two")
+        
+        XCTAssertEqual(["one", "four"], list)
```

Comment:
My first thought was that this API will remove duplicates of an item. It seems that is not true.

Could you refactor the name of this method to removeItemsWithValue?